docs: recommend users install the GitHub or Bitbucket Cloud hosted app (#29457)
renovatebot/renovate: lib/modules/platform/github/readme.md

# GitHub and GitHub Enterprise Server

Most of the information on this page is meant for users who want to self-host Renovate on GitHub or GitHub Enterprise Server.

## Easiest way to run Renovate

> For most users, the easiest way to get started is to install [the Mend Renovate app](https://github.com/marketplace/renovate) from the GitHub marketplace.

Reviewer: ```suggestion
For users on GitHub Cloud (`github.com`), the easiest way to get started is to install [the Mend Renovate app](https://github.com/marketplace/renovate) from the GitHub marketplace.
```

When you use the app, Mend will:

> - authenticate the Renovate app to GitHub or GitHub Enterprise

Reviewer: ```suggestion
- authenticate the Renovate app to GitHub
```

- keep the tokens safe

> - update the Renovate program for you

Reviewer: ```suggestion
- maintain and update the Renovate version used
```

If you self-host Renovate you must do the things listed above yourself.
Self-hosting is meant for users with advanced use cases, or who want to be in full control of the bot and the environment it runs in.
We recommend most users install the Mend Renovate app.

> Read the [Security and Permissions](https://docs.renovatebot.com/security-and-permissions/) page to learn about the Security and Permissions needed for the Mend Renovate app. 

Author: ```suggestion
Read the [Security and Permissions](https://docs.renovatebot.com/security-and-permissions/) page to learn about the Security and Permissions needed for the Mend Renovate app.
```

> After you installed the hosted app, please read the [reading list](./../../../../docs/usage/reading-list.md) to learn how to use and configure Renovate.

Reviewer 2: ```suggestion
After you installed the hosted app, please read the [reading list](../../../reading-list.md) to learn how to use and configure Renovate.
```
